### ISSUE TITLE
Feature Request: Add spell checking and correction in the input box

### ISSUE THREAD

**Issue by SamStrongTalks** [squidowl/halloy#230]
I think the title says it all, I can get screenshots of how this works in some other applications if desired.

**Comment by dmd** [comment]
I think this not a good idea unless you can simply use the OS's text input method.

**Comment by RSully** [comment]
Yeah I think a lot of the input issues (this one, #211, #312, maybe others?) would be solved if we could use native input fields. That would be a huge win IMO

**Comment by johnmmcgee** [comment]
Would be awesome. There was also this ticket, as well: https://github.com/squidowl/halloy/issues/1400
